apps/test/unit/templates/studioHomepages/CourseCardTest.js

```diff
@@ -29,7 +29,11 @@ describe('CourseCard', () => {
     expect(
       wrapper.containsMatchingElement(
         <a href={courses[0].link}>
-          <img />
+          {
```

Comment:
nit: Curious if we want this comment in tests? If the corresponding component does have alt text added, this test will fail but I think that's okay. On the other hand, if someone decides that empty alt text is fine for the corresponding component, they'll likely forget to remove this comment.